scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import yaml, os, shutil
+
+dir = os.path.dirname(__file__)
+
+if os.path.exists('{}/validation_messages'.format(dir)):
+    shutil.rmtree('{}/validation_messages'.format(dir))
```

Comment:
@allait Have updated to use docopt to take an output path as an argument, and also to abort the script if the output already exists.